### ISSUE TITLE
Proximity sensor app crash

### ISSUE THREAD

**Issue by GLWine** [lodauria/AudioSpeedUp#28]
I know I'm looking for the hair in the egg, however if I repeat the following action that is: you start the test and cover the sensor and try to change the speed of the reproduction. In the long run, the app crashes.
Tell me if a video can help you understand the problem

--
Nexus 5x Android 8.1

**Comment by ghost** [comment]
Yes, a video would be useful, I'm not able to trigger this error in any manner. In normal use, does the proximity sensor work well?
